src/pages/Experiments/Experiment/Drawer/ResultsDrawer/ResultsParametersCard.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Card } from 'antd';
+import { LoadingOutlined, PlusOutlined } from '@ant-design/icons';
+import { CommonTable } from 'components';
+
+const ResultsParametersCard = ({ isAdding, parameters, figures }) => {
```

Comment:
I believe that this component is not being used anymore. Check it and remove if unused.